server/src/main/java/org/candlepin/audit/QpidEventMessageReceiver.java

```diff
@@ -57,7 +58,16 @@ public void onMessage(ClientMessage msg) {
             }
 
             // Process the message via our EventListener framework.
-            body = msg.getBodyBuffer().readString();
+            if (msg.getType() == ClientMessage.TEXT_TYPE) {
```

Comment:
Any reason that in this PR the text type handling was added to QpidEventMessageReceiver, but on the master PR it wasn't?

Reply:
That'd be an oversight on my part. I only noticed it in this branch because of the hierarchy change and for whatever reason I didn't think to go back and update master.

Leave this PR up over the weekend as a reminder and I'll get something posted on Monday for master.